bundle/manifests/cache.open-cluster-management.io_ocmsearches.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: ocmsearches.cache.open-cluster-management.io
+spec:
+  group: cache.open-cluster-management.io
```

Comment:
What does the `cache` part means here? I wonder if `observability` would describe our topology better.

_(I describe search as a cache, but there's debate about that. 😃 )_

Reply:
We have not completely defined the CRD . We can remove once we defined the CRD.